CMakeLists.txt

```diff
@@ -401,7 +401,7 @@ if(NOT HAVE_POSITION_INDEPENDENT_CODE AND NOT WIN32)
 endif()
 
 # Useful additional variables that extensions can use.
-if(UNIX AND NOT APPLE)
+if(CMAKE_SYSTEM MATCHES Linux)
```

Comment:
Note: This commit has been integrated as https://github.com/python-cmake-buildsystem/python-cmake-buildsystem/commit/06189e5939ecf49611a996fcff8d96751d118651